### PR TITLE
Date Modify in Post List

### PR DESCRIPTION
**Pull request by cabilangan112**

### CODE REVIEW

File: jassen/project/blog/templates/Post_list.html [hunk 1]
```diff
@@ -7,10 +7,11 @@
       <h1>{{obj.blog.heading}}</h1>
       <h2>{{obj.blog.sub_Heading}}</h2>
       <h3>{{obj.title}}</h3>
-      <strong><p>Posted By:{{obj.author}}</p></strong> 
-      <small> Posted on {{ obj.date|timesince }} ago</small></p>
-      <p><img src="{{ obj.MEDIA_URL }}{{obj.banner_photo.url}}" width="500px"></p>
-      <h3><a href=" "> </a></h3>
+      <p><strong>Posted By:</strong> {{obj.author}}</p>
+      <small> Posted on: {{ obj.date|timesince }} ago</small></p>
+      <p><img src="{{ obj.MEDIA_URL }}{{obj.banner_photo.url}}" width="570px"></p>
```
Comment (rocity) [inline jassen/project/blog/templates/Post_list.html]: @cabilangan112 will the image not show if we remove `{{ obj.MEDIA_URL }}`?

Comment (cabilangan112) [reply jassen/project/blog/templates/Post_list.html]: yes sir.

Comment (cabilangan112) [reply jassen/project/blog/templates/Post_list.html]: mag show japun sir

Comment (cabilangan112) [reply jassen/project/blog/templates/Post_list.html]: gi tangal ko na sir.

**Pull request merged**